README.md

```diff
@@ -28,6 +28,143 @@ Let's simulate a simple arm whose elbow is actuated by a muscle, using
+                       humerus,
+                       osim.Vec3(0, 0.8, 0))
+
+biceps.addNewPathPoint("insertion",
+                       radius,
+                       osim.Vec3(0, 0.7, 0))
+
+# ---------------------------------------------------------------------------
+# Add a controller that specifies the excitation of the muscle.
+# ---------------------------------------------------------------------------
+
+brain = osim.PrescribedController()
+brain.addActuator(biceps)
+brain.prescribeControlForActuator("biceps",
+                                  osim.StepFunction(0.5, 3.0, 0.3, 1.0))
+
+# ---------------------------------------------------------------------------
+# Build model with components created above.
+# ---------------------------------------------------------------------------
+
+arm.addBody(humerus)
+arm.addBody(radius)
+arm.addJoint(shoulder) # Now required in OpenSim4.0
+arm.addJoint(elbow)
+arm.addForce(biceps)
+arm.addController(brain)
+
+# ---------------------------------------------------------------------------
+# Add a console reporter to print the muscle fibre force and elbow angle.
+# ---------------------------------------------------------------------------
+
+# We want to write our simulation results to the console.
+reporter = osim.ConsoleReporter()
+reporter.set_report_time_interval(1.0)
+reporter.addToReport(biceps.getOutput("fiber_force"))
+elbow_coord = elbow.getCoordinate().getOutput("value")
+reporter.addToReport(elbow_coord, "elbow_angle")
+arm.addComponent(reporter)
+
+# ---------------------------------------------------------------------------
+# Add display geometry. 
+# ---------------------------------------------------------------------------
+
+bodyGeometry = osim.Ellipsoid(0.1, 0.5, 0.1)
+bodyGeometry.setColor(osim.Gray)
+humerusCenter = osim.PhysicalOffsetFrame()
+humerusCenter.setName("humerusCenter")
+humerusCenter.setParentFrame(humerus)
+humerusCenter.setOffsetTransform(osim.Transform(osim.Vec3(0, 0.5, 0)))
+humerus.addComponent(humerusCenter)
+humerusCenter.attachGeometry(bodyGeometry.clone())
+
+radiusCenter = osim.PhysicalOffsetFrame()
+radiusCenter.setName("radiusCenter")
+radiusCenter.setParentFrame(radius)
+radiusCenter.setOffsetTransform(osim.Transform(osim.Vec3(0, 0.5, 0)))
+radius.addComponent(radiusCenter)
+radiusCenter.attachGeometry(bodyGeometry.clone())
+
+# ---------------------------------------------------------------------------
+# Configure the model.
+# ---------------------------------------------------------------------------
+
+state = arm.initSystem()
+# Fix the shoulder at its default angle and begin with the elbow flexed.
+shoulder.getCoordinate().setLocked(state, True)
+elbow.getCoordinate().setValue(state, 0.5 * osim.SimTK_PI)
+arm.equilibrateMuscles(state)
+
+# ---------------------------------------------------------------------------
+# Simulate.
+# ---------------------------------------------------------------------------
+
+manager = osim.Manager(arm)
```

Comment:
There are some additional lines here in the C++ version:
```cpp
// Configure the visualizer.
model.updMatterSubsystem().setShowDefaultGeometry(true);
Visualizer& viz = model.updVisualizer().updSimbodyVisualizer();
viz.setBackgroundType(viz.SolidColor);
viz.setBackgroundColor(White);
```
Should the user assume this configuration is done automatically in Python but not in C++?